test_openai.rb

```
@@ -0,0 +1,69 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative 'lib/dialog_ai_assist'
+
+# Configure the gem with OpenAI API key
+DialogAiAssist.configure do |config|
+  # IMPORTANT: Replace with your actual API key or use environment variable
+  config.openai_api_key = ENV['OPENAI_API_KEY'] || 'your-api-key-here'
```

RuboCop (1.76.1)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)